src/test/java/com/znsio/api/posts.feature

```diff
@@ -0,0 +1,13 @@
+@prod @posts
+  Feature: API test for https://jsonplaceholder.typicode.com
+
+    Scenario: Get posts of user
+      Given def userId = 1
+      And def checkUserIdInPosts = karate.call('classpath:com/znsio/templates/postsTemplates.feature@getPostsOfUser', {'userId' : userId}).checkUserIdInPosts
```

Comment:
checkUserIdInPosts incorrect name of the variable. It doesn't represent what it contains.